### PR TITLE
06/27/2017

### PR DESCRIPTION
**Pull request by sfrancis2017**

### CODE REVIEW

**Comment by sfrancis2017** [comment]
06/27/2017

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-git.youphptube.com
\ No newline at end of file
+www.sfrancis.ca
```
Comment (Maikuolan) [inline CNAME]: @DanielnetoDotCom @sfrancis2017

Is this change intentional or accidental (the `CNAME` change)?

**Comment by sfrancis2017** [comment]
Yes..thats right..i do work on this project with Daniel..on private repos. not everything is posted but this one is personally customized..

**Comment by sfrancis2017** [comment]
accidental..it was for my personal repo

**Comment by Maikuolan** [comment]
Okay, no worries. :+1: 

I discovered this project only an hour or so ago, due to a Facebook discussion about it started by Daniel, and so, I've only just started looking through it all. Not accusing anyone of anything or anything like that, of course. I just thought I should ask about this PR, because accidents like this do happen sometimes, and - although I don't know anything about the style of maintaining this project that Daniel follows due to that I discovered it only an hour or so ago - I have encountered a number of projects before where contributions from existing contributors are not thoroughly reviewed due to the existence of previously accepted PRs, and thus, where accidents then end up being accepted into master.

Commenting/asking about it can help to avoid this sometimes in such cases. :-)

**Comment by sfrancis2017** [comment]
Hey man..its a great project.. Kudos to Daniel..I was looking through Cumulusclips to start a personal video sharing space when i came across Daniel's project..i think this one beats it all out there..cheers!!
